src/core/etl/tests/Flow/ETL/Tests/Integration/DataFrame/RenameTest.php

```diff
@@ -98,6 +103,24 @@ public function test_rename_all_to_snake_case() : void
         );
     }
 
+    public function test_rename_all_transliterate() : void
+    {
+        $rows = rows(row(int_entry('ÓSMY', 8)), row(int_entry('DZIEWIĄTY', 9)));
+
+        $ds = df()
+            ->read(from_rows($rows))
+            ->renameEach(rename_transliterate())
+            ->getEachAsArray();
+
+        self::assertEquals(
+            [
+                ['osmy' => 8],
+                ['dziewiaty' => 9],
+            ],
+            \iterator_to_array($ds)
+        );
+    }
+
```

Comment:
dont you also need to replace those deprecated methods in remaining tests with the renemaeEach to avoid static analysis complaining about usage of deprecated methods?

Reply:
Split test + added phpunit specific attribute on deprecated tests.